Raspberian Boxstation Janitorial Hall Changes

BotTierPC opened the pull request:
## About the Pull Request

I have hated Boxstation Janitorial hallway since when I joined! Next to no exageration either, i had made a request for its change back on 12/1/2024 

<img width="1107" height="434" alt="Screenshot 2025-09-23 132925" src="https://github.com/user-attachments/assets/b45d2f8b-3e87-4948-8fc7-f83e795c808b" />

My issue being, ITS PROBLEMATIC! 

<img width="734" height="398" alt="Screenshot 2025-09-23 141340" src="https://github.com/user-attachments/assets/f1b43e22-6a40-49e4-ba78-f832db0ac58f" />

First of all, the light on the Top wall to the Left of the lizard, pretty sure thats broken round start, adding to the problems i have of;
> Janitorial being unnoticable despite in arrivals hallway (Many people have needed to be shown it, and for a new player role, not good)
> The signs! Like WHAT!! WHY ARENT THEY UNDER THE DOORS SO EVERYONE CAN SEE THEM
<img width="487" height="337" alt="Screenshot 2025-09-23 141407" src="https://github.com/user-attachments/assets/b3d87054-074a-4cf8-a895-87dcc7c883a2" />

Anyways with **MAJOR HELP FROM YOONE** (Thank you again!) I took these problems _personally_ and have made this PR as a result

## Why its good for the game

It fixes the problems aforementioned, while adding a little recreational area (with a moth plushie!!)

## Testing Photographs & Evidence
<img width="936" height="424" alt="Screenshot 2025-09-23 140434" src="https://github.com/user-attachments/assets/de0ad38b-4661-4325-a500-2714d860ac9d" />

<img width="793" height="386" alt="Screenshot 2025-09-23 135410" src="https://github.com/user-attachments/assets/abe75c15-5f4a-4a90-9d55-e318da892894" />
<sub>(Moth Plushie not included in picture due to Privacy Concerns)</sub>

## Changelog
:cl:
tweak: Changes to Boxstation Janitorial and Minor Hall Tweak
/:cl:

BotTierPC committed:
Raspberian Boxstation Janitorial Hall Changes

Fixes the DANK and DIRTY hallway that boxstation arrivals>janitoral had, moving a sign and adding a place to sit instead of random tables

mrmanlikesbt commented on _maps/_basemap.dm:
```diff
@@ -1,9 +1,9 @@
 // uncomment this for a map you need to use
 // #define FORCE_MAP "corgstation"
-// #define FORCE_MAP "boxstation"
+ #define FORCE_MAP "boxstation"
```
<img width="251" height="232" alt="Screenshot 2025-08-23 191404" src="https://github.com/user-attachments/assets/4f9f5c59-04d9-43cd-8f72-b590bc888edb" />

BotTierPC replied:
oop

BotTierPC committed:
Update _basemap.dm

BriggsIDP commented on code/__HELPERS/files.dm:
He

BotTierPC replied:
Code is back to being gender fluid i believe, sorry for giving it a gender without the reveal party

BotTierPC committed:
Update files.dm

BotTierPC committed:
Update BoxStation.dmm

BotTierPC committed:
Janitorial Rework

Removes the nearby unused storage room and adds the space into a revamped Janitorial

BotTierPC committed:
Removes not working decal

i tried to add a blood trail dyed black and renamed to look like oil tracks but its just not working as intended so gonzo with it (its already removed in the photo above)

BotTierPC committed:
Removes Black VV from Decal

A decal was dyed black, forgotten about and caused an error, prayers are that this fixes

BotTierPC committed:
Connets the APC to Grid

Adds 3 wires connecting the apc to wire line

BotTierPC committed:
Uses mapping_helpers/airlock... on 3 relevant

The nearby Maintence door, along with the two in janitorial get the helper

BotTierPC committed:
Fixes a error caused by pixels or something

No clue, literally had to turn it on and off again

spockye reviewed (approved):
<img width="854" height="586" alt="image" src="https://github.com/user-attachments/assets/1d504740-f1dc-4d83-97da-6a5c3fc348ac" />

i'd appreciate if you could move this light down to the south wall. im sure everyone has burned their hand more than once when trying to reach at a button 😔 

also a decal inside the wall marked

this is a good improvement, thank you!

mrmanlikesbt reviewed (commented):
should there be a random event spawner on the custodial closet airlock? im not a maptainer so what do i know, but that doesn't seem right

BotTierPC committed:
Minor Changes based on suggested comments

Moved the event marker back INTO janitorial - Moves the light formerly on the wall with lightswitch/button to underneath the locker - Adds a second jani spawn location in the bay - removes decal under a wall
